Chess/state.cpp

```diff
@@ -0,0 +1,130 @@
+#include "state.h"
+
+
+State::State()
+{
+}
+
+State::~State()
+{
+}
+
+
+void State::cellAction(Game *state, QString str){ throw NotImplementedEx(); }
```

Comment:
I have forgotten to ask about it. How should I to proccess this exception? empty return, err messege or what?
I ges it should be something like this, but not sure what to write in catch blocks

void Game::cellAction(QString str)
{
    try {
        state_->cellAction(this, str);
    } catch (NotImplementedEx&) {
        //do what??
    } catch (ChessEx&) {
        //do what??
    } catch (std::exception&) {
        //do what??
    }
}